### ISSUE TITLE
`CryptoJSW.algo.MD5.create` can not use in web worker

### ISSUE THREAD

**Issue by YogiLiu** [originjs/crypto-js-wasm#9]
When use `CryptoJSW.algo.MD5.create` in web worker, it throw the following error.
```
TypeError: CryptoJSW.algo.MD5.create is not a function
```
See the [Code at replit.com](https://replit.com/@lcuff10/crypto-js-wasm-worker#worker.js).

```js
// worker.js
import CryptoJSW from '@originjs/crypto-js-wasm'

self.onmessage = async (e) => {
  await CryptoJSW.algo.MD5.loadWasm()
  const hasher = CryptoJSW.algo.MD5.create()
  hasher.update(e.data)
  const md5 = hasher.finalize()
  const str = CryptoJSW.enc.Base64.stringify(md5)
  postMessage(str)
}
```

```js
// main.js
const el = document.querySelector('#app')
const worker = new Worker(
  new URL('./worker.js', import.meta.url),
  {
    type: 'module',
  },
)

worker.onmessage = (e) => {
  el.innerText = e.data
}

worker.onerror = (e) => {
  el.innerText = e
}

worker.postMessage('hello, world')
```

**Comment by PeterAlfredLee** [comment]
Hi @YogiLiu 
Thank you for your report.

Considering `crypto-js` was created about 15 yrs ago - at which time javascript was not that useable like it is today - `crypto-js` use `create` functions to create instance of a class, which shoule be actually done by constructor of a class.

So we have rewrite `crypto-js` by replacing `create` functions with constructor of each algo class. 

In short words, you can use `new cryptoJsWasm.algo.MD5()` instead of `CryptoJSW.algo.MD5.create()` to get a md5 instance.

**Comment by PeterAlfredLee** [comment]
It's not difficult to implement `create` for md5 and other algos, actually I have merged #2 which is a similar case.

BTW, a PR is alwasy welcome! :)
